client/codegen_wrapped/state_id.cs.tpl

```diff
@@ -4,6 +4,6 @@ public unsafe ContractDataHandle Contract
     {
         var raw_ptr = generated.__Internal.TpClientContractPropertiesStatesStateId{{inner_mangled_name}}Contract(this.Inner.Value.p);
         var ptr = new Ptr<ContractDataHandle>(raw_ptr);
-        return new ContractDataHandle(ptr);
+        return new ContractDataHandle(ptr, OwnershipSemantics.SharedRef);
```

Comment:
The rust code has this boxed, so this must be `Owned`. https://github.com/0xTELEPORTAL/constellation/blob/2f90dae624ebb56d001bc0f52674a2311c89149a/client/rust/src/contract/properties/states/id.rs#L78